source/features/tags-on-commits-list.tsx

```diff
@@ -86,7 +86,7 @@ async function getTags(lastCommit: string, after?: string): Promise<CommitTags>
 			}
 		}
 		`);
-	const nodes: TagNode[] = repository.refs.nodes;
+	const {nodes}: {nodes: TagNode[]} = repository.refs;
```

Comment:
This is one of those lint rules that in this case just ruin the code. Does this work?
```suggestion
	const nodes = repository.refs.nodes as TagNode[];
```

Reply:
Yes